scripts/ec2/ec2_launcher.py

```diff
@@ -57,7 +57,6 @@ def get_parser():
 
     if working_dir is None:
         logger.info('No pending task to be processed. Exit the program.')
-        s.close()
```

Comment:
I don't know much about sqlalchemy, but it's a good practice to use with-statement. For example,

with s.begin_nexted():
    s.do_something()
    s.commit()

I can't find any good example for sql alchemy though. I only found this link/: http://docs.sqlalchemy.org/en/latest/orm/session_transaction.html